pages/linux/factorio.md

```diff
@@ -0,0 +1,12 @@
+# factorio
+
+> Create and start a headless factorio server.
+> More information: <https://wiki.factorio.com/Multiplayer>.
+
+- Create a new save file:
+
+`{{path/to/}}factorio --create {{path/to/save_file.zip}}`
```

Comment:
```suggestion
`{{path/to/factorio}} --create {{path/to/save_file.zip}}`
```
It looks weird when only part is highlighted.

Reply:
![image](https://github.com/tldr-pages/tldr/assets/42655600/e8650b4f-5c8b-46dd-8bad-d0f6898a9b47)
Looks alright to me. Having it be partly highlighted would be correct as the binary name stays the same.